_posts/2023-03-02-q1-survey-results.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+authors:
+  - apasun
+title: "OpenSearch Project Q1 community survey results"
+date: 2023-03-02 
+categories:
+- community
+meta_keywords: OpenSearch NPS, OpenSearch net promoter score, OpenSearch user experience
+meta_description: OpenSearch Project Q1 community survey results, reporting a net promoter score of 57.8 and a UX health score of 77.2.
+
+excerpt: "Thank you to the OpenSearch community members who responded to our Q1 community survey! Our first quarterly survey focused on understanding who our users are, what they use OpenSearch for, and establishing a baseline for how they feel about the current user experience on OpenSearch. We are thrilled to report a net promoter score (NPS) of **57.8** and a UX health score of **77.2** with **97** community members taking part in the survey."
+
+---
+
+Thank you to the OpenSearch community members who responded to our Q1 community survey! Our first quarterly survey focused on understanding who our users are, what they use OpenSearch for, and establishing a baseline for how they feel about the current OpenSearch user experience. We are thrilled to report a net promoter score (NPS) of **57.8** and a UX health score of **77.2**, with **97** community members taking part in the survey. 
+
+We’ve recently launched a research program in order to get a holistic view of of the OpenSearch user experience. The program is designed to evaluate a user’s journey comprehensively and using various research methodologies, including a quarterly survey. In addition to quarterly surveys, we also use "deep dives," which are qualitative research studies that include conversations with users to understand their use cases, requirements, and OpenSearch experience. The quarterly surveys will ask community members to provide feedback on new features, existing OpenSearch experiences, and more. This will help us prioritize improvements to best serve the community. This will help us prioritize improvements to best serve the community. After our Q2 survey is released this month, you will be able to access it at [OpenSearch.org](http://opensearch.org/), as shown in the following image:
```

Comment:
```suggestion
We’ve recently launched a research program in order to get a holistic view of the OpenSearch user experience. The program is designed to evaluate a user’s journey comprehensively using various research methodologies, including a quarterly survey. In addition to quarterly surveys, we also use "deep dives," which are qualitative research studies that include conversations with users to understand their use cases, requirements, and OpenSearch experience. The quarterly surveys will ask community members to provide feedback on new features, existing OpenSearch experiences, and more. This will help us prioritize improvements to best serve the community. After our Q2 survey is released this month, you will be able to access it at [OpenSearch.org](http://opensearch.org/), as shown in the following image.
```